proto/ask_plan.go

```diff
@@ -60,6 +61,16 @@ func (m *AskPlan) Validate() error {
 	return m.GetResources().GetGPU().Validate()
 }
 
+func (m *AskPlan) SaleTime() time.Duration {
```

Comment:
It's not **time** actually, but the duration.

Reply:
yep, I'll rename, but this one is exactly you are looking for.

Reply:
No, it's not, because if I create an ask plan and if it never be placed on the market, it returns 0, while must return `birthTime - now`.